package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "azure-openapi-linter",
-  "version": "0.1.4",
-  "description": "Azure OpenAPI Linter",
+  "name": "@microsoft.azure/azure-openapi-validator",
```

Comment:
Yes, quite intentional, @fearthecowboy renamed all azure-related npm packages to read as ```@microsoft.azure/*``` for consistency. Perhaps we could rename some of our other npm packages accordingly.